pkg/replica/server.go

```diff
@@ -81,11 +73,10 @@ func (s *Server) Open() error {
 	}
 
 	_, info := s.Status()
-	size := s.getSize(info.Size)
 	sectorSize := s.getSectorSize()
 
-	logrus.Infof("Opening volume %s, size %d/%d", s.dir, size, sectorSize)
-	r, err := New(size, sectorSize, s.dir, s.backing, s.revisionCounterDisabled)
+	logrus.Infof("Opening volume %s, size %d/%d", s.dir, info.Size, sectorSize)
```

Comment:
OK, what would happen if the actual volume size is bigger than the backing file size?

Reply:
In that case, it's similar to the volume expansion: the backing file is like an old snapshot with a small size.